src/local.lambda.ts

```diff
@@ -6,26 +6,38 @@ import { LambdaHandler, RequestEvent } from './types';
 
 const DefaultPort = 8000;
 
+// binary upload content-type headers
+const defaultBinaryContentTypeHeaders = [
+  'application/octet-stream',
+  'image/png',
+  'image/jpeg',
+  'image/gif',
+  'application/pdf',
+  'application/zip',
+];
+
 export class LocalLambda {
   handler: LambdaHandler;
   port: number;
   context: Context;
   enableCORS: boolean;
+  binaryContentTypesOverride: string[];
```

Comment:
Minor thing here, `Set` might be a better fit here. 
we can still ask users to pass in array in the config, but convert to Set internally in this class. 
Not a big deal but should be a bit more efficient I guess.

Reply:
Addressed it. Please review it and let me know if there are any other concerns. Thanks.

Reply:
Addressed it. Please let me know if there are any other concerns. Thanks for your review @vcfvct .